features/03_testing_frameworks/cucumber/steps/filesystem/append_to_file.feature

```diff
@@ -26,7 +26,7 @@ Feature: Append content to file
     When I run `cucumber`
     Then the features should all pass
 
-  Scenario: Append to a non-existing file
+  Scenario: Append to a non-existing file (deprecated)
```

Comment:
Question (not a sticking point)

Is this the format we'll be doing in features for deprecated stuff? We can copy to other repos if desired.

Reply:
I think it's clear this way, but maybe there are other options.